bower.json

```diff
@@ -13,6 +13,7 @@
     "o-assets": ">=0.4.1 <4",
     "o-viewport": ">=2.2.0 <4",
     "o-icons": ">=4.4.2 <6",
-    "o-spacing": "^2.0.0"
+    "o-spacing": "^2.0.0",
+    "o-brand": "^3.2.5"
```

Comment:
I think we should add this to MIGRATION.md

Reply:
I'll update all our MIGRATION.md when we do the final rollout at the same time :)